packages/reports/addon/components/dimension-selector.js

```diff
@@ -118,15 +119,30 @@ export default Component.extend({
       }, {});
   }),
 
+  /**
+   * @property {Boolean} enableRequestPreview
+   */
+  get enableRequestPreview() {
```

Comment:
Do we need to add this as a prop for the component?

Reply:
We use it in the template

Reply:
Ya, we have a helper that can fetch a feature flag for the template. I felt it doesn't really make sense as a property in a component, since its not associated with the component in any way.

Reply:
Oh wasn't aware of the helper, will switch

Reply:
done